examples/ipyleaflet.ipynb

```diff
@@ -0,0 +1,248 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import micropip\n",
+    "\n",
+    "await micropip.install(['bqplot', 'ipyleaflet'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from urllib.request import urlopen\n",
+    "import json\n",
+    "from datetime import datetime\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from ipywidgets import Dropdown\n",
+    "\n",
+    "from bqplot import Lines, Figure, LinearScale, DateScale, Axis\n",
+    "\n",
+    "from ipyleaflet import Map, GeoJSON, WidgetControl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = pd.read_json('https://raw.githubusercontent.com/jupyter-widgets/ipyleaflet/master/examples/nations.json')"
```

Comment:
We probably still need to use the `from js import fetch` trick to fetch extra data for now:

https://github.com/jtpio/jupyterlite/issues/119#issuecomment-854495013